packages/app/src/server/routes/apiv3/import.js

```diff
@@ -355,7 +355,7 @@ export default function route(crowi) {
       return res.apiv3(data);
     }
     catch {
-      const msg = 'the version of this growi and the growi that exported the data are not met';
+      const msg = 'the version of this GROWI and the GROWI that exported the data are not met';
       const varidationErr = 'versions-are-not-met';
       return res.apiv3Err(new ErrorV3(msg, varidationErr), 500);
```

Comment:
```suggestion
      const validationErr = 'versions-are-not-met';
      return res.apiv3Err(new ErrorV3(msg, varldationErr), 500);
```
`validationErr` の key って i18n の key ではないのか？
kebab-case と snake_case が混じってる。

Reply:
これはフロント側でエラーコードで処理を変えたりするためのエラーコード的なやつだと思います。